tests/integration/sourcemap_monkey_integ_test.py

```diff
@@ -0,0 +1,133 @@
+from configparser import ConfigParser
+from importlib import import_module
+from pathlib import Path
+from unittest import mock
+
+import pytest
+
+"""
+This file monkey-patches ConfigParser in order to enable source mapping
+and test the results of source mapping various PyTeal dapps.
+"""
+
+
+BRUTE_FORCE_TERRIBLE_SKIPPING = (
+    """The second router is flaky due to issue 199, so skipping for now"""
+)
```

Comment:
TODO: can I turn this skipping off yet?

Reply:
yes?

Reply:
This I would re-investigate after pr #600 gets merged... but it's unlikely to be unskippable.

Reply:
I've added a task to #681, and am resolving this:

<img width="659" alt="image" src="https://user-images.githubusercontent.com/291133/221606172-4a3af79f-bfb5-45f3-b759-83c267a2e215.png">